src/Components/Charts/Bar.tsx

```diff
@@ -0,0 +1,387 @@
+import React, { useContext, useEffect, useState } from "react";
+import { ResponsiveBar } from '@nivo/bar'
+import axios from "axios";
+import { AuthContext } from "../../Context/AuthContext";
+
+import { ResponsiveStream } from '@nivo/stream'
+export default function Bar() {
+  const { baseUrl, requstHeaders }: any = useContext(AuthContext);
+  const [contTasks, setContTasks] = useState([]);
+  const [contUsers, setcontUsers] = useState([]);
+  const getContTasks = () => {
```

Comment:
There is a typo here in `getContTasks`, it should be `getCountTasks`. The same goes for the state variables.